src/bindings/c/docs/OpenVINO_Runtime_C_API_User_Guide.md

```diff
@@ -0,0 +1,62 @@
+# User Guide of OpenVINO C API
```

Comment:
It is not a right place for user documentation. User documentation should be located inside the docs folder.